.github/workflows/DeployFunction.yml

```diff
@@ -0,0 +1,41 @@
+name: Deploy DotNet project to Azure Function App
+
+on:
+  workflow_call:
+    inputs:
+      function-app-name: 
+        type: string
+        required: true
+      function-project-path:
+        type: string
+        required: true
+
+env:
+  DOTNET_VERSION: '8.0.x'
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - name: 'Checkout GitHub Action'
+      uses: actions/checkout@v3
+
+    - name: Setup DotNet ${{ env.DOTNET_VERSION }} Environment
+      uses: actions/setup-dotnet@v3
+      with:
+        dotnet-version: ${{ env.DOTNET_VERSION }}
```

Comment:
Ikke fullstendig relatert til den PRen her, men skulle vi kanskje sett på å bruke global config i alle workflowsene? Ser vi at har definert 8.x i BuildAndTest.yml. 

https://github.com/actions/setup-dotnet?tab=readme-ov-file#using-the-global-json-file-input

https://learn.microsoft.com/en-us/dotnet/core/tools/global-json

Hadde vært nice å låse versjonene også til exact kanskje? 🤔

Hva tycks?

Reply:
Tar en prat om det i morgen 👍🏼